# Publications

1. <i class="ai ai-biorxiv ai-2x"/> J.A. Cartes, P. Bonizzoni, S. Ciccolella, G. Della Vedova, L. Denti, X. Didelot, D.C. Monti, Y. Pirola\\
**RecGraph: adding recombinations to sequence-to-graph alignment**\\
*Bioinformatics (2024)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1093/bioinformatics/btaa779" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-biorxiv ai-2x"/>  J.A. Cartes, P. Bonizzoni, S. Ciccolella, G. Della Vedova, L. Denti\\
**PangeBlocks: customized construction of pangenome graphs via maximal blocks**\\
*bioRxiv (2024)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1101/2024.09.17.613426" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-biorxiv ai-2x"/>  S. Mahadevaraju, S. Pal, P. Bhaskar, B.D. McDonald, L. Benner, L. Denti, D. Cozzi, P. Bonizzoni, T.M. Przytycka, B. Oliver\\
**Diverse somatic Transformer and sex chromosome karyotype pathways regulate gene expression in Drosophila gonad development**\\
*bioRxiv (2024)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1101/2024.08.12.607556" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-closed-access ai-2x"/> L. Denti\*, P. Khorsand\*, P. Bonizzoni, F. Hormozdiari, R. Chikhi\\
**SVDSS: structural variation discovery in hard-to-call genomic regions using sample-specific strings from accurate long reads**\\
*Nature Methods (2023)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1038/s41592-022-01674-1" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> D. Cozzi, P. Bonizzoni, L. Denti\\
**ESGq: Alternative Splicing Events Quantification across Conditions based on Event Splicing Graphs**\\
*ITAT Proceedings (2023)* <a class="ai ai-ceur ai-lg" href="https://ceur-ws.org/Vol-3498/paper31.pdf" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-biorxiv ai-2x"/>  S. Ciccolella, D. Cozzi, G. Della Vedova, S. Kuria, P. Bonizzoni, L. Denti\\
**Differential quantification of alternative splicing events on spliced pangenome graphs**\\
*bioRxiv (2023)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1101/2023.11.06.565751" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> S. Ciccolella\*, L. Denti\*, P. Bonizzoni, G. Della Vedova, Y. Pirola\*, and M. Previtali\*\\
**MALVIRUS: an integrated web application for viral variant calling**\\
*BMC Bioinformatics (2022)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1186/s12859-022-04668-0" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-springer ai-2x"/> G. Bernardini\*, L. Denti, M. Previtali\\
**Alignment-Free Genotyping of Known Variations with MALVA**\\
*Methods in Molecular Biology (2022)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1007/978-1-0716-2293-3_15" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> P. Khorsand\*, L. Denti\*, Human Genome Structural Variant Consortium, P. Bonizzoni, R. Chikhi, F. Hormozdiari\\
**Comparative genome analysis using sample-specific string detection in accurate long reads**\\
*Bioinformatics Advances (2021)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1093/bioadv/vbab005" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> L. Denti\*, Y. Pirola\*, M. Previtali\*, T. Ceccato, G. Della Vedova, R. Rizzi, P. Bonizzoni\\
**Shark: fishing relevant reads in an RNA-Seq sample**\\
*Bioinformatics (2020)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1093/bioinformatics/btaa779" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> S. Ciccolella, G. Bernardini, P. Bonizzoni, M. Previtali, G. Della Vedova\\
**Triplet-based similarity score for fully multi-labeled trees with poly-occurring labels**\\
*Bioinformatics (2020)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1093/bioinformatics/btaa676" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> L. Denti\*, M. Previtali\*, G. Bernardini, A. Schönhuth, P. Bonizzoni\\
**MALVA: genotyping by Mapping-free ALlele detection of known VAriants**\\
*Iscience (2019)*  <a class="ai ai-doi ai-lg" href="https://doi.org/10.1016/j.isci.2019.07.011" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-elsevier ai-2x"/> S. Beretta, L. Denti, M. Previtali\\
**Graph Theory and Definitions**\\
*Encyclopedia of Bioinformatics and Computational Biology (2019)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1016/B978-0-12-809633-8.20421-4" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-elsevier ai-2x"/> S. Beretta, L. Denti, M. Previtali\\
**Network Properties**\\
*Encyclopedia of Bioinformatics and Computational Biology (2019)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1016/B978-0-12-809633-8.20422-6" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-open-access ai-2x"/> L. Denti, R. Rizzi, S. Beretta, G. Della Vedova, M. Previtali, P. Bonizzoni\\
**ASGAL: aligning RNA-Seq data to a splicing graph to detect novel alternative splicing events**\\
*BMC Bioinformatics (2018)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1186/s12859-018-2436-3" style="text-decoration:none; color:inherit;"/>

1. <i class="ai ai-springer ai-2x"/>  S. Beretta, P. Bonizzoni, L. Denti, M. Previtali, R. Rizzi\\
**Mapping RNA-seq data to a transcript graph via approximate pattern matching to a hypertext**\\
*AlCoB (2017)* <a class="ai ai-doi ai-lg" href="https://doi.org/10.1007/978-3-319-58163-7_3" style="text-decoration:none; color:inherit;"/>
{: reversed="reversed"}

<br/>

# Talks

1. **ESGq: Alternative Splicing Events Quantification across Conditions based on Event Splicing Graphs**\\
*ITAT23. Tatranské Matliare, Slovakia*

1. **SVDSS: structural variation discovery in hard-to-call genomic regions using sample-specific strings from accurate long reads**\\
*RECOMB2023 - Instanbul, Turkey*

1. **Structural Variations Discovery from Specific Strings**\\
*Nov. 9, 2022. Comenius University (invited by Tomas Vinar)*

1. **Comparative genome analysis using sample-specific string detection in accurate long reads**\\
*HiTSeq2021 - Lyon, France (virtual)*

1. **Comparative genome analysis using sample-specific string detection in accurate long reads**\\
*DSB2021 - Milano, Italy (virtual)*

1. **Dynamic quasi-minimal perfect hash function for k-mers** *(work-in-progress talk)*\\
*DSB2020 - Rennes, France*


1. **MALVA: genotyping by Mapping-free ALlele detection of known VAriants**\\
*BITS2019 - Palermo, Italy*

1. **MALVA: genotyping by Mapping-free ALlele detection of known VAriants**\\
*RECOMBSEQ2019 - Washington DC, USA*

1. **Mapping RNA-seq data to a transcript graph via approximate pattern matching to a hypertext**\\
*AlCoB2017 - Aveiro, Portugal*
{: reversed="reversed"}

<br/>

# Posters
1. **MALVA: genotyping by Mapping-free ALlele detection of known VAriants** <a class="fa fa-file-pdf-o" href="/pdfs/malva_recomb2019.pdf" style="text-decoration:none; color:inherit;"></a>\\
*RECOMB2019*

1. **ASGAL: aligning RNA-Seq data to a splicing graph to detect novel alternative splicing events** <a class="fa fa-file-pdf-o" href="/pdfs/asgal_recomb2018.pdf" style="text-decoration:none; color:inherit;"></a>\\
*RECOMB2018* <a href="/imgs/award_recomb2018.jpg" style="text-decoration:none; color:inherit;"><i class="fa fa-trophy" style="color:#b5e853" aria-hidden="true"></i> <i style="color:#b5e853">Best Poster Award</i></a>
{: reversed="reversed"}
